src/highcharts-ng.js

```diff
@@ -50,26 +51,74 @@ if (typeof module !== 'undefined' && typeof exports !== 'undefined' && module.ex
       }, 0, false);
     };
 
+        this.removeUnlinkedObjects = function(mergedConfig) {
+            /*
+                Removes unlinked objects, items that have been removed in the config,
+                but not yet removed from the HighChart object
+            */
+
+            //First check to see if there are any yAxis that need to be removed
+            //If a series is linked to the yAxis, it will be removed by HighCharts
+
+            if (mergedConfig.yAxis && Array.isArray(mergedConfig.yAxis)) {
```

Comment:
@pablojim I made the fiddle. This is my first time contributing to open source in a public way, so please let me know if everything looks good. The only change I can think of is the CDN file may need to change from my PR to the master repo when merged with master